### ISSUE TITLE
outlier status incorrectly applied to integrated age

### ISSUE THREAD

**Issue by mjz1983** [NMGRL/pychron#2104]
When making data tables, outlier status (in the attached example, outlier on isochron) is being applied to the total integrated age. The total integrated age should include all steps.
[SMCN-01 hbl table.xlsx](https://github.com/NMGRL/pychron/files/7936306/SMCN-01.hbl.table.xlsx)

**Comment by jirhiker** [comment]
This is a feature. you specify how the total integrated age should be calculated, all steps or all valid steps

**Comment by mjz1983** [comment]
Ah. I might need a quick hand to show me how to turn on/off this feature.

Matt Zimmerer
Field Geologist II
New Mexico Bureau of Geology
New Mexico Tech
801 Leroy Pl
Socorro, NM 87801
***@***.***
575.835.5114

**Comment by mjz1983** [comment]
When the outlier steps are included in the total integrated age, those
outlier steps are also used in the isochron calculation, which they
shouldn't.

Is it possible to generate a data table where steps are omitted on the
isochron, but include in the total integrated age?

MZ

**Comment by jirhiker** [comment]
The expected behavior is 

- Total Integrated will use all steps whether they are omitted or not. 
- Valid Integrated uses only the non-omitted steps

**Comment by cburrill** [comment]
For the tables I think what we need is an option (check box) to use either the Total Integrated or Valid Integrated Age. Currently under the summary rows section, there is only the option for the Total Integrated age. So if steps are marked invalid and they are included on the table, they are currently being used in the integrated age and you need to change the integrated age manually or not include the invalid steps at all. I think the preference would be to show the invalid steps on the table but not have them included in any of the age calculations.

![image](https://user-images.githubusercontent.com/58988891/152207246-4713d053-555b-4238-b2e0-982190edd0c9.png)

![image](https://user-images.githubusercontent.com/58988891/152209769-f6655023-acbc-4114-862f-5b73fd29e4c9.png)

![image](https://user-images.githubusercontent.com/58988891/152209608-fe9ca5ce-9b67-41c8-a058-57c1fb3c3053.png)

**Comment by jirhiker** [comment]
Don't know if this addresses your issue but the preferred age kind (Total Integrated, Valid Integrated) is set at the Group Age Node. 

<img width="1580" alt="Screen Shot 2022-02-02 at 4 46 48 PM" src="https://user-images.githubusercontent.com/2035568/152256865-7f77e956-24b3-4242-8152-c98e531d8f04.png">

**Comment by cburrill** [comment]
No, the correct option to change it isn't there. The age row changes whats in the summary where the plateau age goes. Even though I have "plateau else valid integrated age" in all the rows, it still shows "total integrated age" in the summary.



![image](https://user-images.githubusercontent.com/58988891/152261347-ed96e31a-8f79-49b6-8c2b-7217f768c398.png)
